apps/site/assets/ts/schedule/components/line-diagram/SingleStop.tsx

```diff
@@ -143,12 +143,19 @@ const StopBranchLabel = (stop: RouteStop): JSX.Element | null =>
   ) : null;
 
 const StopGraphic = (isOrigin = false, isTerminus = false): JSX.Element => {
-  let yPosition = "-32px";
-  if (isTerminus) {
-    yPosition = isOrigin ? "-66px" : "32px";
+  // this hardcoded position will be shown in IE, otherwise overwritten by CSS
+  let yPosition = "32px";
+  if (isTerminus && isOrigin) {
+    yPosition = "-3px";
   }
   return (
-    <svg viewBox="0 10 10 10" className="m-schedule-diagram__line-stop">
+    <svg
+      viewBox="0 10 10 10"
+      preserveAspectRatio="xMidYMin slice"
+      width="100%"
+      height="10px"
+      className="m-schedule-diagram__line-stop"
+    >
       <circle r={`${isTerminus ? "5" : "4"}`} cx="50%" cy={yPosition} />
```

Comment:
Re: `yPosition` and the `r` attribute, I think we should consistently either have both of the conditionals as ternaries here, or both of them as variable assignments beforehand. IMO the conditions are short enough that ternaries would be perfectly readable so, no strong opinions here 🙂